### DISCUSSION TITLE
Oscillating noise in analog output

### DISCUSSION THREAD

**Discussion by jrfloeder** [pyPhotometry/code#20] [Q&A] [answered]
Hello! We’ve been using pyphotometry for a while without issues, but today we noticed constantly oscillating noise in the analog output of our system. We tried unplugging and reconnecting everything, including the BNC cables and USB, but the noise remained. We also did a factory reset of the filesystem without success in removing the noise. Surprisingly, we noticed that the noisy oscillations got bigger when we opened Google chrome and returned to normal when we minimized it (blue bracket in image 1). Furthermore, the oscillations flattened when we used the snip & sketch tool to make a screenshot (blue bracket in image 2). Perhaps this suggests that the noise may be coming from some software issue? Any help with this matter would be greatly appreciated :)

![oscillations_chrome_open](https://github.com/user-attachments/assets/5a67875f-ccf8-4c23-a2ac-e76b940d4026)
![oscillations_screenshot_snip sketch](https://github.com/user-attachments/assets/f1fd023b-645b-4b22-906c-348be21869c4)

**Comment by ThomasAkam** [comment]
Hmmm, that is wierd.  What does the signal look like if you unplug the LEDs and the photoreciever from the pyPhotometry board and aquire signal in continuous mode?

**Reply by jrfloeder** [reply, marked as answer]
Thank you for your suggestion! When we unplugged the LEDs & photoreceiver, the signal in continuous mode was artifact-free. With a bit more investigation, we identified a faulty power adaptor as the issue. So, with the adaptor switched out, everything is resolved :)
